services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: datetime.date = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        date_obj = datetime.strptime(session_date, "%Y-%m-%d")
+        queryset = queryset.filter(show_time__date=date_obj.date())
```

Comment:
you can simplify this - orm will take care of filtering by date object without conversion